packages/console/src/pages/SignInExperience/components/SignInMethodsForm.tsx

```diff
@@ -26,6 +27,16 @@ const SignInMethodsForm = () => {
 
   useEffect(() => {
     if (primaryMethod) {
+      if (oldPrimaryMethod.current) {
+        // The secondary sign in method should select the old primary method by default.
+        setValue(`signInMethods.${oldPrimaryMethod.current}`, true);
+        // eslint-disable-next-line @silverhand/fp/no-mutation
+        oldPrimaryMethod.current = undefined;
```

Comment:
this can be omitted, as you are setting the new value at line 38

Reply:
updated.